modules/router/routes/users.js

```diff
@@ -27,4 +28,42 @@ router.get('/api/getUser/:user', async (ctx) => {
   };
 });
 
-module.exports = router;
+// Registracija novega uporabnika.
+router.post('/api/registerUser', async (ctx) => {
```

Comment:
`ctx` je vbistvu krajše za context. Prav si nardil, to je vbistvu celotna zahteva, objekt, ki se pošlje na server. To vsebuje vse kar request vsebuje: headerje, query parametre, URL, body itd.